src/client/mod.rs

```diff
@@ -341,6 +341,36 @@ impl Client {
         self.net().0.load().addresses()
     }
 
+    /// Returns number of max node attempts
```

Comment:
```suggestion
    /// Returns the max number of times a node can be retried before removing it from the network.
```
If the wording I gave is a little off, feel free to fix, but the general gist.